spec/lcm/integration/fixtures/user_provisioning_fixtures.rb

```diff
@@ -0,0 +1,76 @@
+require_relative '../support/user_provisioning_helper'
+require_relative '../support/configuration_helper'
+require_relative '../support/connection_helper'
+require_relative '../support/s3_helper'
+require_relative '../support/lcm_helper'
+require_relative '../support/project_helper'
+require_relative 'base_fixtures'
+require_relative 'project_fixtures'
+
+module Fixtures
+  class UserProvisioningFixtures < BaseFixtures
+    FIXTURE_ID_PREFIX += 'userprov-e2e-'
+
+    def initialize(opts = {})
+      user_amount = opts[:user_amount] || 2
+      rest_client = LcmConnectionHelper.production_server_connection
+      domain = rest_client.domain(LcmConnectionHelper.environment[:prod_organization])
+
+      users = Array.new(user_amount) do |n|
+        {
+          custom_login: "lcm-userprov-e2e-UPPERCASE-#{n}@gooddata.com",
+          position: 'developer'
+        }
+      end
+
+      project_fixtures = ProjectFixtures.new domain: domain, rest_client: rest_client, project_amount: 2
+
+      user_data = project_fixtures[:clients].map do |c, p|
```

Comment:
info: Refers to 'project_fixtures' more than self (maybe move it to another class?) (FeatureEnvy) (reek)